andela/activo-web
# #159569304 Default loading message category

**What does this PR do?**
Adds a default loader to the categories list page while asset categories are being fetched

**Description of Task to be completed?**

- set default loader for categories list page
 - use the spinner component created in #159569285

**How should this be manually tested?**
After cloning, run the app by running `yarn start:dev`
- After navigating to the categories list page, You should see a simple spinner loading while the assets are being fetched.

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**

[#159569304](https://www.pivotaltracker.com/story/show/159569304)

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2018-08-07 at 3 30 26 pm" src="https://user-images.githubusercontent.com/29452947/43782257-d7ccf120-9a56-11e8-96fd-597e8ad6ad1c.png">

<img width="1430" alt="screen shot 2018-08-07 at 3 30 07 pm" src="https://user-images.githubusercontent.com/29452947/43782277-e229a9f6-9a56-11e8-8f52-bf25d02ca031.png">


**Questions:**

N/A
